config/locales/en.yml

```diff
@@ -16,11 +16,18 @@
 #
 # This would use the information in config/locales/es.yml.
 #
+# The following keys must be escaped otherwise they will not be retrieved by
+# the default I18n backend:
+#
+# true, false, on, off, yes, no
+#
+# Instead, surround them with single quotes.
+#
+# en:
+#   'true': 'foo'
+#
 # To learn more, please read the Rails Internationalization guide
 # available at http://guides.rubyonrails.org/i18n.html.
 
 en:
-  activerecord:
-    attributes:
-      user:
-        is_fallback: Fallback
```

Comment:
In short: sometimes you want your column name to say one thing for the sake of developer clarity, but be called another thing when viewed for the sake of user clarity. The locales file provides a central place to define what those 'translations' should look like. Rails then uses them by default to do things like `label`s, `human_attribute_name`s, etc.

More details at http://guides.rubyonrails.org/i18n.html.